kernel-api/src/main/scala/org/apache/toree/interpreter/broker/BrokerState.scala

```diff
@@ -69,10 +72,21 @@ class BrokerState(private val maxQueuedCode: Int) {
     // Add the code to be executed to our queue and the promise to our map
     codeQueue.add(brokerCode)
     promiseMap.put(brokerPromise.codeId, brokerPromise)
-
+    // Maintain an output stream for each codeId
+    outputResultStreamMap.put(brokerPromise.codeId, outputResultStream)
     codeExecutionPromise.future
   }
 
+  // Called from pyspark
```

Comment:
Can you provide a real Scaladoc for this versus "Called from pyspark"? Something more descriptive, but not specific to pyspark either since it's in the generic `BrokerState` class.